From 5ccd006dc8fe7a012950e0c0be7902135886cbb1 Mon Sep 17 00:00:00 2001
From: Daniel Bratell <bratell@opera.com>
Date: Fri, 21 Jul 2017 10:01:24 +0000
Subject: [PATCH] Fix compilation for ATK (accessibility)

A variable wasn't given the right name inside the ATK support when
a patch consolidated 3 AX properties into single restrictions
property for accessibility objects.

R=aleventhal@chromium.org, jochen@chromium.org

Change-Id: I0c82e055e39614110cfc4f7a7299d226bd4e6ad2
Reviewed-on: https://chromium-review.googlesource.com/580927
Reviewed-by: Jochen Eisinger <jochen@chromium.org>
Commit-Queue: Daniel Bratell <bratell@opera.com>
Cr-Commit-Position: refs/heads/master@{#488630}
---
 content/browser/accessibility/browser_accessibility_auralinux.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/content/browser/accessibility/browser_accessibility_auralinux.cc b/content/browser/accessibility/browser_accessibility_auralinux.cc
index 9caccec3edbe..11d709daf89b 100644
--- a/content/browser/accessibility/browser_accessibility_auralinux.cc
+++ b/content/browser/accessibility/browser_accessibility_auralinux.cc
@@ -571,7 +571,7 @@ static AtkStateSet* browser_accessibility_ref_state_set(AtkObject* atk_object) {
 // it's best to leave this out rather than break people's builds:
 #if defined(ATK_CHECK_VERSION)
 #if ATK_CHECK_VERSION(2, 16, 0)
-      atk_state_set_add_state(atk_state_set, ATK_STATE_READ_ONLY);
+      atk_state_set_add_state(state_set, ATK_STATE_READ_ONLY);
 #endif
 #endif
       break;
-- 
2.11.0

